### ISSUE TITLE
I want to specify UIRectCorner on RoundCornerImageProcessor

### ISSUE THREAD

**Issue by 131e55** [onevcat/Kingfisher#668]
#### What

Hi😃 Thank you for developing a very useful library❗

I want to specify `UIRectCorner` on `RoundCornerImageProcessor` and `Image.kf.image`.

For example
```swift
let processor = RoundCornerImageProcessor(cornerRadius: 20, targetSize: CGSize(width: 40, height: 40), byRoundingCorners: [.topRight, .bottomRight])
```
Would you please add this?

#### Other Comment
I have read the `3.7.1` source code for `Image.kf.imgae`.
If iOS, just specify a parameter of `UIBezierPath`.
But if macOS, I dont know how to specify...

**Comment by onevcat** [comment]
Good suggestion. I will look at it now to see what we can do.

**Comment by onevcat** [comment]
Implemented in #669

**Comment by 131e55** [comment]
Thank you very much @onevcat 😍❗
